src/components/FicheFilm/FicheFilm.jsx

```diff
@@ -74,17 +105,37 @@ export default function FicheFilm() {
     UsersScorePictures = star1;
   }
 
-  const hour = (runtimes - (runtimes % 60)) / 60;
-  const min = (runtimes / 60 - hour) * 60;
+
+  useEffect(() => {
+    localStorage.setItem('watchlist', JSON.stringify(watchlist));
+  }, [watchlist]);
+
+  let UsersScorePictures;
+  if (voteAverage === 10) {
+    UsersScorePictures = star5;
+  } else if (voteAverage < 10 && voteAverage >= 8) {
+    UsersScorePictures = star4;
+  } else if (voteAverage < 8 && voteAverage >= 6) {
+    UsersScorePictures = star3;
+  } else if (voteAverage < 6 && voteAverage >= 4) {
+    UsersScorePictures = star2;
+  } else if (voteAverage < 4) {
+    UsersScorePictures = star1;
+  }
+
+  const hour = (runtime - (runtime % 60)) / 60;
+  const min = (runtime / 60 - hour) * 60;
```

Comment:
`const min = runtime % 60;` (Yeah, this `%` operator is *really* useful :p )

Reply:
I added corrections!